search-api/src/main/scala/no/ndla/searchapi/service/search/MultiSearchService.scala

```diff
@@ -170,7 +164,11 @@ trait MultiSearchService {
       val articleTypeFilter = Some(
         boolQuery().should(settings.articleTypes.map(articleType => termQuery("articleType", articleType)))
       )
-      val taxonomyContextTypeFilter   = contextTypeFilter(settings.learningResourceTypes)
+      val learningResourceTypeFilter = Some(
+        boolQuery().should(
+          settings.learningResourceTypes.map(resourceType => termQuery("learningResourceType", resourceType.entryName))
+        )
+      )
```

Comment:
Er det ikke litt kult å beholde funksjonaliteten for at denne ikke genererer noe query dersom lista er tom? 

Utesta, men kanskje noe sånt?:
```suggestion
      val learningResourceTypeFilter = Option.when(settings.learningResourceTypes.nonEmpty)(
        boolQuery().should(
          settings.learningResourceTypes.map(resourceType => termQuery("learningResourceType", resourceType.entryName))
        )
      )
```